### PR TITLE
Add product listing page

### PR DESCRIPTION
**Pull request by yash9263**

### CODE REVIEW

**Review by iambizan — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -104,17 +104,21 @@ <h1 class="header-1">Skis</h1>
     <footer class="main-footer">
       <div class="footer-item shop-links">
         <h3 class="header-2">Shop</h3>
-        <a class="small-text">Men</a>
-        <a class="small-text">Ladies</a>
```
Comment (MayankKumar10) [inline index.html:108]: just nitpicking instead of ladies use Woman

File: styles/pages/products.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
+.wrapper {
+  display: grid;
+  grid-template-columns: 240px 1fr;
```
Comment (MayankKumar10) [inline styles/pages/products.css:3]: use rem instead of px

**Review by MayankKumar10 — APPROVED** [review]
Everything else Looks good you can merge it.

**Review by Maianki — APPROVED** [review]
Looks good. Just one suggestion you can add ```target=_blank ``` in the social media links you have provided in the footer.

File: styles/pages/products.css (hunk 2 above)
Comment (yash9263) [reply styles/pages/products.css:3]: Don't want to grow it beyond 240px that's why using px

Comment (MayankKumar10) [reply styles/pages/products.css:3]: That's Okay then.

File: index.js [hunk 3]
```diff
@@ -1 +1,47 @@
...
+        />
+        <div class="wishlist-icon"><i class="bx bxs-heart"></i></div>
+    </div>
+    <div class="card-content">
+        <div class="small-text product-caption">Trending</div>
+        <h3 class="header-3 bold">${title}</h3>
+        <strong class="small-text bold">Rs.1,200.00</strong>
+        <div class="small-text medium">New Arrival</div>
+    </div>
+</div>
```
Comment (manojsarna) [inline index.js:39]: good execution

File: styles/base/global.css [hunk 4]
```diff
@@ -3,7 +3,7 @@
 }
 
 body {
-  background: var(--color-primary-100);
+  background: var(--color-warning-100);
   min-height: 100%;
   height: unset;
 }
```
Comment (manojsarna) [inline styles/base/global.css:9]: good color coding!!

File: styles/pages/products.css (hunk 2 above)
Comment (manojsarna) [inline styles/pages/products.css:3]: you can use rems over here

File: styles/pages/products.css [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+.aside-filters {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  padding: 16px 16px 32px;
+  position: sticky;
+  top: 12px;
+  transform: scale(0.9);
```
Comment (manojsarna) [inline styles/pages/products.css:21]: can use rems

File: styles/pages/products.css [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+.aside-section {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+}
+.aside-section h2 {
+  padding-bottom: 12px;
+}
+.aritcle-input-wrapper label {
+  margin-left: 4px;
```
Comment (manojsarna) [inline styles/pages/products.css:44]: can use rems

File: styles/pages/products.css [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+  padding: 24px 0px;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 24px 8px;
+}
+
+@media screen and (max-width: 68.75rem) {
+  .products {
+    grid-template-columns: repeat(3, 1fr);
+  }
```
Comment (manojsarna) [inline styles/pages/products.css:59]: check for rems

**Review by manojsarna — APPROVED** [review]
Overall its very well written

good to merge

File: styles/pages/products.css (hunk 7 above)
Comment (yash9263) [reply styles/pages/products.css:59]: rem hi hai

File: styles/pages/products.css (hunk 2 above)
Comment (yash9263) [reply styles/pages/products.css:3]: don't want to the column to increase beyond 240px that's why using px

File: styles/base/global.css (hunk 4 above)
Comment (yash9263) [reply styles/base/global.css:9]: thanks

File: index.js (hunk 3 above)
Comment (yash9263) [reply index.js:39]: thanks

File: index.html [hunk 8]
```diff
@@ -104,17 +104,21 @@ <h1 class="header-1">Skis</h1>
...
-        <a class="small-text">Kids</a>
-        <a class="small-text">Accesories</a>
+        <ul>
+          <li><a class="small-text">Men</a></li>
+          <li><a class="small-text">Ladies</a></li>
+          <li><a class="small-text">Kids</a></li>
```
[the remark this answers is not in the document]
Comment (yash9263) [reply index.html:110]: thanks

**Review by pankaj-prog — APPROVED** [review]
code is clean and readable,  Great work.

**Pull request merged**